docs/app/globals.css

```diff
@@ -1,26 +1,51 @@
-:root {
-  --page-padding: 0.75rem;
-  --page-padding-md: 2rem;
-  --page-padding-xl: 3rem;
-
-  --page-header-height: 56px;
-
-  --page-sub-nav-absolute-width: 252px;
-  --page-sub-nav-width: calc(
-    var(--page-sub-nav-absolute-width) + var(--page-padding-md)
-  );
-  --page-sub-nav-width-xl: calc(
-    var(--page-sub-nav-absolute-width) + var(--page-padding-xl)
-  );
-  --page-sub-nav-padding: 2.25rem;
-  --page-sub-nav-padding-xl: 2.5rem;
-
-  --page-main-padding: 5rem;
-  --page-main-padding-xl: 5.5rem;
-
-  --pre-padding-x: 1rem;
-}
+@import 'tailwindcss';
+@import './../node_modules/@marigold/theme-docs/src/theme.css';
+
+@import './../node_modules/@marigold/theme-core/dist/styles.css';
+@import './../node_modules/@marigold/theme-b2b/dist/styles.css';
+
+/* demos */
+@source './../content/**/*.{js,ts,jsx,tsx,mdx}';
+
+/* from docs */
+@source './../ui/**/*.{js,ts,jsx,tsx,mdx}';
+@source './**/*.{js,ts,jsx,tsx,mdx}';
+
+@source './../../themes/theme-docs/src/**/*.ts';
+
+/* marigold components */
+@source './../../packages/{components,system}/**/*.{tsx,ts}';
+@source '!../../packages/{components,system}/node_modules/**/*.{tsx,ts}';
+@source '!../../packages/{components,system}/**/*.{stories,test}.{tsx,ts}';
+
+/* tailwind plugins */
 
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@plugin '@tailwindcss/typography';
+@plugin 'tailwindcss-animate';
+/* https://github.com/adoxography/tailwind-scrollbar/issues/104 */
+@plugin 'tailwind-scrollbar';
+
+@layer base {
+  :root {
```

Comment:
put these in `@theme`?

Reply:
they were before also in globals.css and are only for the components inside the docs relevant